taxi/urls.py

```diff
@@ -7,6 +7,13 @@
     DriverListView,
     DriverDetailView,
     ManufacturerListView,
+    CarCreateView,
+    CarUpdateView,
+    CarDeleteView,
+    ManufacturerCreateView,
+    ManufacturerUpdateView,
+    ManufacturerDeleteView,
```

Comment:
Consider grouping imports using parentheses `()` as per the checklist requirement. This will improve readability and maintain consistency with the code style guide.

Reply:
here and so imports are grouped  using parentheses ()